app/templates/components/file-actions.emblem

```diff
@@ -17,6 +17,17 @@
             | Generate Report
           a.margin-left href=ireneFilePath target="_blank" class="button is-primary"
             | Visit Dashboard
+          span.dropdown.dropdown--default.margin-left
+            = basic-dropdown horizontalPosition='right' as |dropdown|
```

Comment:
I think we can use `attach-tooltip` here instead of custom drop down

something like `= attach-tooltip placement="top"`

We are doing similar thing during logout

Reply:
We would require dropdown menu here which has a clickable action, tooltip is not what we needed here.